src/casp/bq_scripts/precalculate_fields.py

```diff
@@ -0,0 +1,30 @@
+import typing as t
+
+from google.cloud import bigquery
+
+from casp.services.utils import get_google_service_credentials
+
+SQL_PRECALCULATE_TOTAL_MRNA_UMIS_FORMAT = """
+    UPDATE `{project}.{dataset}.cas_cell_info` ci
+    SET ci.total_mrna_umis = m.raw_counts_total
+    FROM (
+        SELECT cas_cell_index, SUM(raw_counts) AS raw_counts_total
+        FROM `{project}.{dataset}.cas_raw_count_matrix` m
+        JOIN `{project}.{dataset}.cas_feature_info` fi ON (fi.cas_feature_index = m.cas_feature_index)
+        WHERE fi.feature_biotype = 'gene'
```

Comment:
Let's put `gene` in an appropriate "constants" python file and replace it with `MRNA_FEATURE_BIOTYPE_NAME`.